src/Board/BoardService.php

```diff
@@ -83,6 +83,26 @@ public function getBoardIssues($id, $paramArray = []): ?\ArrayObject
         }
     }
 
+    /**
+     * @return \ArrayObject|AgileIssue[]|null
+     */
+    public function getBoardBacklogIssues($id, $paramArray = []): ?\ArrayObject
+    {
+        $json = $this->exec($this->uri.'/'.$id.'/backlog'.$this->toHttpQueryParameter($paramArray), null);
+
+        try {
+            return $this->json_mapper->mapArray(
+                json_decode($json, false, 512, JSON_THROW_ON_ERROR)->issues,
```

Comment:
JSON_THROW_ON_ERROR constant only supported by PHP 7.3 or above.
but compose.json defined required php 7.1 or above.

Reply:
Trying to avoid exactly this issue, I copied `getBoardIssues` method, renamed it, and changed the endpoint being hit from `/issue` to `/backlog`. 

This same issue shows up in every method in this file on `master`.

Reply:
ok i will be modifying it.